examples/composite_queries/src/canister2/canister2.py

```diff
@@ -0,0 +1,43 @@
+from kybra import Async, CanisterResult, ic, manual, nat, Principal, query, update
+from src.canister1.types import StringQueryResult
+from src.canister3.types import Canister3
+
+canister3 = Canister3(Principal.from_str('r7inp-6aaaa-aaaaa-aaabq-cai'))
+
+counter: nat = 0
+
+
+@query
+def inc_counter() -> nat:
+    global counter
```

Comment:
global should not be necessary here

Reply:
It is necessary without it pylance gives a '"counter" is unbound' error